RELEASENOTES.md

```diff
@@ -10,6 +10,21 @@ have unified the code into a single branch, and made the AWS-specific parts a
 compile-time option.  When a feature (or entire release) only supports one of
 the two variants, we note that in the release notes.
 
+# v1.12.1-aws
+
+All users of v1.12.0-aws are strongly recommended to take this fix when using
+EFA Installer >= 1.35.0.
+
+Bug fixes:
+* platform-aws vf sorting code produces significant performance regressions
```

Comment:
or crashes.  If asserts are turned on in NCCL, 1.12.0 will crash with NCCL 2.23.

Reply:
ack, will amend